docs/fastdds/dds_layer/topic/contentFilteredTopic/writerFiltering.rst

```diff
@@ -6,3 +6,24 @@
 Where is filtering applied: writer vs reader side
 =================================================
 
+Content filters may be evaluated on either side, as the DataWriter obtains the filter expression from the
+DataReader during discovery.
+
+Filtering on the writer side can save network bandwidth at the cost of increasing CPU usage on the writer.
+
+.. _dds_layer_topic_contentFilteredTopic_writer_restrictions:
+
+Conditions for writer side filtering
+------------------------------------
+
+A DataWriter will perform filter evaluation for a DataReader when all of the following conditions are met.
+Filtering will otherwise be performed by the DataReader.
+
+- The DataWriter has infinite liveliness. See |LivelinessQosPolicy|.
+- Communication with the DataReader is neither intra-process nor data-sharing.
```

Comment:
Similarly to the previous point maybe a link can be added into `intra-process` and `data-sharing` words.